src/Nelmio/Alice/Instances/Processor/Processable.php

```diff
@@ -45,6 +45,12 @@ public function getValue()
     public function valueMatches($regex)
     {
         if (preg_match($regex, $this->value, $matches)) {
+            $matches       = array_map(
```

Comment:
I'll remove this, it will result in a failing nightly branch for now. When PHP decides to be BC again the branch should turn green (on a rebuild)